src/App.js

```diff
@@ -1,13 +1,84 @@
+import { Routes, Route, Link } from 'react-router-dom';
 import './App.css';
 import Calculator from './components/Calculator';
 import './components/Calculator.css';
 
 function App() {
   return (
-    <div className="App">
-      <Calculator />
+    <div>
+      <div className="top-bar">
+        <h1>Math Magicians</h1>
+        <nav className="links">
+          <Link to="/">Home</Link>
+          <Link to="/Calculator">Calculator</Link>
+          <Link to="/quote">Quote</Link>
+        </nav>
+      </div>
+
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="quote" element={<Quote />} />
+        <Route path="calculator" element={<CalculatorPage />} />
+      </Routes>
+      <div className="footer">
+        <p>&copy; Math magicians 2022</p>
+      </div>
     </div>
   );
 }
 
+function Home() {
+  return (
+    <>
+      <main className="main-container">
+        <h2>Welcome to Our page!</h2>
+        <p>
+          It is a long established fact that a reader will be distracted by the
+          readable content of a page when looking at its layout. The point of
+          using Lorem Ipsum is that it has a more-or-less normal distribution of
+          letters, as opposed to using Content here, content here, making it
+          look like readable English.
+        </p>
+        <p>
+          It is a long established fact that a reader will be distracted by the
+          readable content of a page when looking at its layout. The point of
+          using Lorem Ipsum is that it has a more-or-less normal distribution of
+          letters, as opposed to using Content here, content here, making it
+          look like readable English.
+        </p>
+      </main>
+    </>
+  );
+}
```

Comment:
- [ ][optional] it would be  to put every page into its own file that will make the code clean and also you will follow the convention of Reactjs